jms/src/main/scala/akka/stream/alpakka/jms/JmsConsumerSettings.scala

```diff
@@ -206,7 +222,7 @@ object JmsConsumerSettings {
   /**
    * Java API: Reads from the default config provided by the actor system at `alpakka.jms.consumer`.
    *
-   * @param actorSystem The actor system
+   * @param actorSystem       The actor system
```

Comment:
I think your IDE is doing some reformatting.

Reply:
Yes, I've run compile and scalafmtAll from  sbt and it seems to be ok with this formatting. Would you rather i just restore the file as it was before?

Reply:
If the code style sbt task is happy then I guess it's fine. I just wanted to point it out in case you had IDE formatting things enabled.